app/assets/stylesheets/pages/home/_subscribe.scss

```diff
@@ -17,6 +17,18 @@
 		line-height: 2;
 		font-weight: normal;
 		margin: 0;
+
+		@media only screen and (max-width: 900px){
```

Comment:
try to change the font when it's hit 768px and 480px

Reply:
okay, already to change